Tests

janis-jenny opened the pull request:
- [ ] Add tests

janis-jenny committed:
install jest

janis-jenny committed:
congifure jest test and watch in package.js

janis-jenny committed:
add babel.config, fix jest installation and add a todos test

janis-jenny committed:
complete the test for 2 objects w a moch of one

janis-jenny committed:
add tests for the button's event in the forms

janis-jenny committed:
add more tests to render.test and project.test

janis-jenny committed:
fix linters

mke2111 reviewed (changes requested):
# STATUS: CHANGES REQUIRED 🔴 ♻️ 

## Hey Happy Coder, 👋🏽 
**This is a great project you have going on, well-done**. 

### Well-done 💯 
    1. The tests run just fine and almost all methods have been tested.
    2. The linter checks pass successfully.

**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

### REQUIRED

- [ ] You have pieces of commented-out code in your test files. Clean up these files.
- [ ] Add the testing command to your readme file

### OPTIONAL

- [ ] You might want to add snapshot tests. These will automatically test the functionality of your code.

#

### Kindly make all the requested changes and submit another CRR. 💯 

**Cheers 🍹 and Happy coding 💻** 

<hr>

Mention @mke2111 in this thread and I'll be glad to help. 😄 
### Best regards
Slack 🆘  Roy Mukuye
Twitter 🐦  [Roy Mukuye](https://twitter.com/Roymkenya)
LinkedIn 🔗  [Roy Mukuye](https://www.linkedin.com/in/roy-mukuye/)

mke2111 commented on src/_tests/render.test.js:
```diff
@@ -0,0 +1,51 @@
+
+/* import {
+  createProject, createTodo, displayProjects, displayTasks,
+} from '../components/render';
```
Remove the commented code from your files

janis-jenny committed:
add the TSE requeritments
Add the testing command in readme file,
Delete commented-out code in the  test files

rukundoeric reviewed (approved):
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project **Jenny**. 🎆 Keep it up._

- ### To Highlight
  - Follows gitwork-flow. 💯 
  - Codebase look clean and professinal. 💯 
  - All tests are passing 💯
  
- ### To Improve
  - N/A

_Congratulations on the splendid job **@janis-jenny** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**
